.circleci/config.yml

```diff
@@ -593,11 +601,7 @@ jobs:
       - run:
           name: "Brew: Tap wix/brew"
           command: brew tap wix/brew
-      - run:
```

Comment:
This was just added in a PR (https://github.com/facebook/react-native/pull/36192), but it seems the python installation is no longer causing CI tests to break. Instead, this `brew unlink` command was throwing an error.